tests/test_processing_algebra.py

```diff
@@ -0,0 +1,111 @@
+import numpy as np
+
+from pyomeca import Markers
+from tests._constants import MARKERS_DATA, ANALOGS_DATA, EXPECTED_VALUES
+from tests.utils import is_expected_array
+
+
+def test_proc_abs():
+    is_expected_array(ANALOGS_DATA.meca.abs(), **EXPECTED_VALUES[1])
+    is_expected_array(MARKERS_DATA.meca.abs(), **EXPECTED_VALUES[2])
+
+
+def test_proc_matmul():
+    m = MARKERS_DATA.sel(axis="x", channel="PSISl").meca.matmul(
+        MARKERS_DATA.sel(axis="y", channel="PSISl")
+    )
+    m_ref = MARKERS_DATA.sel(axis="x", channel="PSISl") @ MARKERS_DATA.sel(
+        axis="y", channel="PSISl"
+    )
+    np.testing.assert_array_almost_equal(m, 87604420.17968313, decimal=6)
+    np.testing.assert_array_almost_equal(m, m_ref, decimal=6)
+
+    a = ANALOGS_DATA.sel(channel="EMG1").meca.matmul(ANALOGS_DATA.sel(channel="EMG10"))
+    a_ref = ANALOGS_DATA.sel(channel="EMG1") @ ANALOGS_DATA.sel(channel="EMG10")
+    np.testing.assert_array_almost_equal(a, -1.41424996e-06, decimal=6)
+    np.testing.assert_array_almost_equal(a, a_ref, decimal=6)
```

Comment:
I am not sure I would test here with real data... I think it would make sense to test with randomly created data (with know seed) and test the whole matrix output?

Reply:
Done in https://github.com/pyomeca/pyomeca/pull/88/commits/f76a559c4cf9d090593cb2bdd9c3819ab7cf7f64.